<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersTab</class>
 <widget class="QWidget" name="UsersTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <item>
    <widget class="QTableView" name="tableUserList">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="buttonNewUser">
       <property name="text">
        <string>New User</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonEditUser">
       <property name="text">
        <string>Edit User</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRemoveUser">
       <property name="text">
        <string>Remove User</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string>Next Login Attempt</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkUsernamePasswordError">
          <property name="text">
           <string>Username/password error:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="editUsernamePasswordError">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Username/password error.</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="checkFreeUserError">
          <property name="text">
           <string>Free user error</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkTFARequired">
          <property name="text">
           <string>2FA required</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkTFAError">
          <property name="text">
           <string>2FA error</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkTFAAbort">
          <property name="text">
           <string>2FA abort</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkTwoPasswordsRequired">
          <property name="text">
           <string>2nd password required</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkTwoPasswordsError">
          <property name="text">
           <string>2nd password error</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkTwoPasswordsAbort">
          <property name="text">
           <string>2nd password abort</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkAlreadyLoggedIn">
          <property name="text">
           <string>Already logged in</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonNewUser</tabstop>
  <tabstop>buttonEditUser</tabstop>
  <tabstop>buttonRemoveUser</tabstop>
  <tabstop>tableUserList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
